INN/umbrella-caribbean
# inner-circle and inner-outline mixins for borders

## Changes

This pull request makes the following changes:

- Recreates `Largo_Byline` class as `Caribbean_Byline` so that we can:
   - show the avatar on more than just single posts
   - wrap the avatar in a div so we can style it because of https://github.com/INN/umbrella-caribbean/issues/3#issuecomment-557684551
- Creates `carribean_byline` function to use the above class
- Creates new `inner-outline` mixin that will add a white outline to buttons, squares, etc.
- Creates new `inner-circle` mixin that will also add a white _rounded_ outline to images
- Styles byline avatar, but it's not actually used anywhere yet

`inner-outline`:
![Screen Shot 2019-11-22 at 4 05 16 PM](https://user-images.githubusercontent.com/18353636/69460344-e9840480-0d41-11ea-83f1-f9ff8718ccd9.png)

`inner-circle` (on an avatar):
![Screen Shot 2019-11-22 at 4 05 25 PM](https://user-images.githubusercontent.com/18353636/69460343-e9840480-0d41-11ea-94fc-355d89eae853.png)

## Why



For #3

## Testing/Questions

Features that this PR affects:

- Bylines and `inner-outline` styling


Questions that need to be answered before merging:

- [ ] Is there a better way to do the circle outline?
- [ ] Can any fat be trimmed from the duplicated `Largo_Byline` as `Caribbean_Byline` class?
- [ ] Not a question, but a note that I included a `content-single.php` template in this PR so that `carribean_byline` can be seen/tested in action

Steps to test this PR:

1. Add the `inner-outline` class/mixin to a button to make sure it looks ok
2. Look at a single post byline avatar and make sure that it looks ok
